### DISCUSSION TITLE
Missing 5 million places with AWS and Azure

### DISCUSSION THREAD

**Discussion by callumscoby** [OvertureMaps/data#100] [Discussion]
Hi all,

I'm using the latest Overture places release (2023-11-14) but I'm only getting 51,267,260 places. In the previous release (2023-10-19) there were ~57 million places, and in the release notes for the current release, it states that there should still be around ~57 million places. 

This occurs when using duckdb and AWS:

`duckdb.sql(f"""
INSERT INTO table
       WITH filtered_data AS (
SELECT
    id,
    REPLACE(JSON_EXTRACT(names, '$.common[0].value'), '"', '') AS poi_name,
    geometry,
FROM
    read_parquet('s3://overturemaps-us-west-2/release/{overture_release}-alpha.0/theme=places/type=*/*', filename=true, hive_partitioning=1))
SELECT id, poi_name, ST_GeomFromWKB(geometry) as geometry
FROM 'filtered_data';
""")`

And when locally downloading the 26 Parquet files from Azure and reading them all in:

`duckdb.sql(f"""
INSERT INTO table
       WITH filtered_data AS (
SELECT
    id 
    REPLACE(JSON_EXTRACT(names, '$.common[0].value'), '"', '') AS poi_name,
    geometry,
FROM
    '*.parquet')
SELECT id, poi_name, ST_GeomFromWKB(geometry) as geometry
FROM 'filtered_data';
""")`

Are there only actually ~51 million places in this release or is there an error in my approach? Any information would be greatly appreciated!

**Comment by jwass** [comment]
@callumscoby 

I ran this in duckdb and it looks like the 57M are all there from AWS. Or are you saying this is just an Azure issue?
```
D select count(*) from read_parquet('s3://overturemaps-us-west-2/release/2023-11-14-alpha.0/theme=places/type=place/*.parquet');
100% ▕████████████████████████████████████████████████████████████▏
┌──────────────┐
│ count_star() │
│    int64     │
├──────────────┤
│     57512517 │
└──────────────┘
D
```

Or maybe your download didn't get all the files?

**Reply by callumscoby** [reply]
I had the issue with both AWS and locally downloading the files in Azure Storage Explorer. I'll give your code a go, thank you!
